use-case-1.groovy

```diff
@@ -0,0 +1,77 @@
+import hudson.model.*
+import jenkins.model.Jenkins
+import java.io.File;
+import groovy.xml.XmlUtil
+import static javax.xml.xpath.XPathConstants.*
+import java.io.InputStream;
+import java.io.FileInputStream
+import javax.xml.transform.stream.StreamSource
+import groovy.util.XmlParser
+
+def startingPoints=["Most_Granular_Level_Of_Jenkins_Folder"]
+def excludedPath = ["Project_3"]
+
+def allProject = []
+def excludedProjects = []
+
+Jenkins.instance.getAllItems(AbstractProject.class).each{item -> 
+  def startingPoint = startingPoints.each{ start -> 
+    if(item.fullName.toLowerCase().startsWith(start.toLowerCase())){
+      allProject.add(item.fullName.toLowerCase())
+    }
+  }
+}
+
+Jenkins.instance.getAllItems(AbstractProject.class).each{item -> 
+  def startingPoint = excludedPath.each{ start -> 
+    if(item.fullName.toLowerCase().contains(start.toLowerCase())){
+      excludedProjects.add(item.fullName.toLowerCase())
+    }
+  }
+}
+
+allProject.intersect(excludedProjects).each{allProject.remove(it);excludedProjects.remove(it)}
+
+def includedProject= allProject+excludedProjects
+
+def nexusIqBuilder = '''
+<org.sonatype.nexus.ci.iq.IqPolicyEvaluatorBuildStep plugin="nexus-jenkins-plugin@3.9.20200722-164144.e3a1be0">
+<com__sonatype__nexus__ci__iq__IqPolicyEvaluator____iqStage>Build</com__sonatype__nexus__ci__iq__IqPolicyEvaluator____iqStage>
+<com__sonatype__nexus__ci__iq__IqPolicyEvaluator____iqApplication class="org.sonatype.nexus.ci.iq.SelectedApplication">
+<applicationId></applicationId>
+</com__sonatype__nexus__ci__iq__IqPolicyEvaluator____iqApplication>
+<com__sonatype__nexus__ci__iq__IqPolicyEvaluator____failBuildOnNetworkError>false</com__sonatype__nexus__ci__iq__IqPolicyEvaluator____failBuildOnNetworkError>
+<com__sonatype__nexus__ci__iq__IqPolicyEvaluator____jobCredentialsId></com__sonatype__nexus__ci__iq__IqPolicyEvaluator____jobCredentialsId>
+<advancedProperties></advancedProperties>
+</org.sonatype.nexus.ci.iq.IqPolicyEvaluatorBuildStep>
```

Comment:
Also better if we can put that in seperate file so that we can read file on groovy instead of in-line xml.

Reply:
XML snippet has been indented for readability and it cannot be referenced from a separate file since this script is to be run from Jenkins script console.